nucypher-core-wasm/src/lib.rs

```diff
@@ -275,16 +322,6 @@ impl MessageKit {
         self.0.conditions.clone().map(Conditions)
     }
 
-    #[wasm_bindgen(js_name = fromBytes)]
-    pub fn from_bytes(data: &[u8]) -> Result<MessageKit, Error> {
-        from_bytes::<_, nucypher_core::MessageKit>(data)
-    }
-
-    #[wasm_bindgen(js_name = toBytes)]
-    pub fn to_bytes(&self) -> Box<[u8]> {
-        to_bytes(self)
```

Comment:
That's a typo. I think normally `clippy` would have caught this, but `&self` was inside the macro. This isn't an issue since the resulting `&&self` would be coerced to just `&self`, but it's worth fixing.